src/chrome/index.js

```diff
@@ -1,17 +1,18 @@
-import { REDUCERS } from '../constants';
 import { Socket } from 'phoenix';
 import { startPlugin } from '..';
 import { setWorkerProfile } from '../actions';
 import listenMessages from './listenMessages';
-import { logDebug } from '../logging';
 import handleWorkerStateNotifications from './handleWorkerStateNotifications';
 import handleWork from './handleWork';
 import handleStateSaving from './handleStateSaving';
 import renderIcon from './renderIcon';
 import startIdleChecking from './startIdleChecking';
+import { applyMiddleware } from 'redux';
+import { logMiddleware } from '../logging';
 
-export const startChromePlugin = (auth, pollUrl, chrome, enhancer, socketConstructor = Socket) => {
```

Comment:
Not really, originally it was gonna be Redux DevTools but that doesn't work with other plugins. I just never really cleaned up.